heartbeat/monitors/task.go

```diff
@@ -166,3 +133,44 @@ func (t *configuredJob) Stop() {
 		t.client.Close()
 	}
 }
+
+func runPublishJob(job jobs.Job, client beat.Client) []scheduler.TaskFunc {
+	event := &beat.Event{
+		Fields: common.MapStr{},
+	}
+	next, err := job(event)
+	hasContinuations := len(next) > 0
```

Comment:
Was asking myself why the error check does not come before this one. It does not matter as there is no return, it just "felt" not 100% clean.

Looking at the usage of `hasContinuations` this could even be after 148?